src/components/Comments/WriteComment.js

```diff
@@ -199,7 +198,7 @@ export const styles = StyleSheet.create({
     flex: 1
   },
   btn: (isDisableSubmit) => ({
-    backgroundColor: !isDisableSubmit ? colors.bondi_blue : '#f2f2f2',
+    backgroundColor: !isDisableSubmit ? COLORS.holyTosca : COLORS.concrete,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the `btn` style has been updated. Ensure that these new colors are consistent with your design system.
```diff
- backgroundColor: !isDisableSubmit ? colors.bondi_blue : '#f2f2f2',
+ backgroundColor: !isDisableSubmit ? COLORS.holyTosca : COLORS.concrete,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->